integreat_cms/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -173,7 +173,7 @@ <h1 class="heading">
                 {% if view.kwargs.url_filter == 'ignored' %}
                     <option value="unignore"
                             data-bulk-action="{% url_for_current_region 'linkcheck' request url_filter=view.kwargs.url_filter %}{{ pagination_params }}">
-                        {% translate "Unignore" %}
+                        {% translate "Bring back to check" %}
```

Comment:
I think this could be confusing, because it is possible to mark e.g. valid links as ignored. unignoring them will cause them to end up at the verified tab, not the unchecked tab.

Maybe we could label it "invalidate" or "revoke"?

Reply:
Nice naming 👍

Reply:
Hmm - I find that quite difficult tbh...maybe "revoke verification"? To make it clearer what we are revoking? And in German "Verifizierung entziehen"? Or "Verifizierung rückgängig machen"?